Makefile

```diff
@@ -112,6 +102,34 @@ ifeq ($(TARGET_ARCH),arm64)
 endif
 endif
 
+ifeq ($(TARGET_OS),windows)
+ifeq ($(SOURCE_OS),linux)
+    ifeq ($(shell which x86_64-w64-mingw32-gcc > /dev/null; echo $$?), 1)
+        $(info MinGW cross compiler not found, installing...)
+        $(shell sudo apt-get update && sudo apt-get install -y mingw-w64)
+    endif
+endif
+ifeq ($(TARGET_ARCH),amd64)
+    GO_TAGS ?= -tags no_cgo
+    # We need the go build command to think it's in cgo mode
+    export CGO_ENABLED = 1
+    export CXXFLAGS := -pthread
+    export CGO_CXXFLAGS := -pthread
+    export CC=/usr/bin/x86_64-w64-mingw32-gcc
+    export CXX=/usr/bin/x86_64-w64-mingw32-g++
+    export AS=x86_64-w64-mingw32-as
+    export AR=x86_64-w64-mingw32-ar
+    export RANLIB=x86_64-w64-mingw32-ranlib
+    export LD=x86_64-w64-mingw32-ld
+    export STRIP=x86_64-w64-mingw32-strip
+    FFMPEG_OPTS += --target-os=mingw32 \
```

Comment:
Doesn't the `32` refer to `32 bit`? Don't we want `64 bit`?

Reply:
It is a bit confusing, but `mingw32` os tag does not indicate 32 bit but rather GNU for Windows.

mingw32 is still commonly used naming convention even though ti has since been forked and supports 64 bit.

Reply:
TODO: add comment about mingw32 actually being 64 bit to avoid confusion.